### PR TITLE
adding ctrlstack integration manifest

### PR DESCRIPTION
**Pull request by jj-ctrlstack**
**Requirements**

- [x] I have followed the repository's [pull request submission guidelines](../blob/main/README.md#submitting-pull-requests)
- [x] I have updated my user documentation
- [ ] I have updated my developer documentation (my integration README)
- [x] I have submitted the [Become a Partner](https://launchdarkly.com/partners/) form

**Intended availability**

Partner: CtrlStack

**Related issues**

N/A

**Describe the solution you've provided**

CtrlStack is a change intelligence platform for better observability. With its ability to observe system changes and data relationships across the stack, CtrlStack provides DevOps teams contextual understanding of system issues for faster incident investigation. Add LaunchDarkly feature flag events to CtrlStack to know when an alert or change in data behavior is directly or indirectly caused by toggling of a feature flag.

**Describe alternatives you've considered**

N/A

**Provide a request preview**

N/A

**Additional context**

N/A

### CODE REVIEW

File: integrations/ctrlstack/manifest.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "name": "CtrlStack",
+  "version": "1.0.0",
+  "overview": "Add LaunchDarkly feature flag events to CtrlStack as change events on a timeline.",
+  "description": "To set this up, attach LaunchDarkly's API access token, and the project and environment IDs in CtrlStack integration settings.",
+  "details": "CtrlStack is a change intelligence platform for better observability. With its ability to observe system changes and data relationships across the stack, CtrlStack provides DevOps teams contextual understanding of system issues for faster incident investigation. Add LaunchDarkly feature flag events to CtrlStack to know when an alert or change in data behavior is directly or indirectly caused by toggling of a feature flag.",
```
Comment (InTheCloudDan) [inline integrations/ctrlstack/manifest.json:6]:
```suggestion
  "details": "CtrlStack is a change intelligence platform for better observability. With its ability to observe system changes and data relationships across the stack, CtrlStack provides DevOps teams contextual understanding of system issues for faster incident investigation.<br><br>Add LaunchDarkly feature flag events to CtrlStack to know when an alert or change in data behavior is directly or indirectly caused by toggling of a feature flag.",
```

File: integrations/ctrlstack/manifest.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  "author": "CtrlStack",
+  "supportEmail": "support@ctrlstack.com",
+  "links": {
+    "site": "https://ctrlstack.com",
+    "supportWebsite": "https://www.ctrlstack.com/contact/",
+    "launchdarklyDocs": "https://docs.launchdarkly.com/integrations/ctrlstack",
+    "privacyPolicy": "https://www.ctrlstack.com/terms-and-policy/"
+  },
+  "categories": ["monitoring", "developer-tools", "automation"],
```
Comment (InTheCloudDan) [inline integrations/ctrlstack/manifest.json:15]:
```suggestion
  "categories": ["monitoring", "developer-tools"],
```

**Review by InTheCloudDan — CHANGES_REQUESTED** [review]
@jj-ctrlstack if we can get these 2 small changes done this looks to get merged, thanks!

**Comment by jj-ctrlstack** [comment]
@InTheCloudDan done! Thanks

**Review by InTheCloudDan — APPROVED** [review]

**Pull request merged**